### ISSUE TITLE
Unrecognized target type for plugin: lv2_sep

### ISSUE THREAD

**Issue by rty65tt** [DISTRHO/DPF#449]
```
$ cmake.exe --version
cmake version 3.28.3

$cmake.exe -G "MinGW Makefiles"  ..
-- The C compiler identification is GNU 13.2.0
-- The CXX compiler identification is GNU 13.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/msys64/ucrt64/bin/cc.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: C:/msys64/ucrt64/bin/c++.exe - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: C:/msys64/ucrt64/bin/pkg-config.exe (found version "2.1.1")
-- Checking for module 'cairo'
--   Found cairo, version 1.18.0
-- Checking for module 'cairo'
--   Found cairo, version 1.18.0
-- Found OpenGL: opengl32
-- Checking for module 'cairo'
--   Found cairo, version 1.18.0
-- Checking for module 'liblo'
--   Found liblo, version 0.32
-- Checking for module 'sdl2'
--   Found sdl2, version 2.30.0
CMake Error at cmake/DPF-plugin.cmake:198 (message):
  Unrecognized target type for plugin: lv2_sep
Call Stack (most recent call first):
  examples/CairoUI/CMakeLists.txt:4 (dpf_add_plugin)


-- Configuring incomplete, errors occurred!

```
What tools should I use for different platforms [win, mac, linux]?
How build?

**Comment by falkTX** [comment]
cmake doesnt use lv2_sep, just lv2. the lv2_sep is for makefiles as it needs a different target for monolithic builds and dsp/ui split builds.

**Comment by rty65tt** [comment]
I just:
```
git clone --recursive https://github.com/DISTRHO/DPF.git
mkdir build
cd build/
cmake.exe -G "MinGW Makefiles"  ..
```
or
``` 
cmake.exe -G "Visual Studio 17 2022" ..
```
and see error:
```
CMake Error at cmake/DPF-plugin.cmake:198 (message):
  Unrecognized target type for plugin: lv2_sep
Call Stack (most recent call first):
  examples/CairoUI/CMakeLists.txt:4 (dpf_add_plugin)
```
Is this how it should be? I expect something else?
I have Windows 10 Visual Studio 17 2022 and msys64/mingw-w64 ucrt.

**Comment by falkTX** [comment]
ah yeah that is a typo on https://github.com/DISTRHO/DPF/blob/main/examples/CairoUI/CMakeLists.txt#L5
it should be lv2 instead of lv2_sep. somehow the CI checks never caught that as a problem

**Comment by rty65tt** [comment]
Visual Studio 17 2022 building success.
Mingw-w64 not.
Thanks.

**Comment by falkTX** [comment]
FYI this was fixed in 6a62ed994029a40b759c0f40f685d00b740ae3b0

**Comment by rty65tt** [comment]
Mingw builing success.  I was wrong yesterday.
